src/functionaltests/java/com/ericsson/ei/notifications/trigger/SubscriptionNotificationSteps.java

```diff
@@ -195,16 +195,16 @@ public void send_one_previous_event() throws Throwable {
         eventManager.sendEiffelEvents(EIFFEL_EVENTS_JSON_PATH, eventNamesToSend);
     }
 
-    @Then("^Default values mail notification are added$")
-    public void default_values_mail_notification() {
-        if (emailSender.getSender() != null && emailSender.getSubject() != null) {
-            assertEquals("noreply@ericsson.com", emailSender.getSender());
-            assertEquals("Email Subscription Notification", emailSender.getSubject());
-        } else if (emailSender.getSender() != null && emailSender.getSubject() == null) {
-            assertEquals("noreply@ericsson.com", emailSender.getSender());
-        } else if (emailSender.getSender() == null && emailSender.getSubject() != null) {
-            assertEquals("Email Subscription Notification", emailSender.getSubject());
-        }
+    @Then("^Then Notification email contains <our expected> values")
+    public void notification_email_contains_expected_values() {
+        sender = "noreply@ericsson.com";
```

Comment:
This doesn't contain any assert. You can send in the two String values from the feature file and then assert that both these values are set in the notification:

assertEquals("expected subject string", emailSender.getSubject())